Makefile

```diff
@@ -237,7 +238,12 @@ CMAKE_FILES+= \
 endif
 
 #----------------------------------------------------------------------------------------------
-BOOST_DIR ?= $(ROOT)/.install/boost
+ifeq ($(wildcard $(ROOT)/.install/boost),)
```

Comment:
we still want to allow passing an explicit `BOOST_DIR` value no? Should we add the condition that it's not defined?

Reply:
oops